app/reports/students_spreadsheet.rb

```diff
@@ -23,14 +23,26 @@ def csv_string(students)
   # the hash should included keys for all fields, even if the student doesn't
   # have them.  In other words, this method is responsible for returning
   # the same shape for all students it is provided.
-  def flat_row_hash(student, all_service_types)
-    student_fields = student.as_json.except(*[
-      'created_at',
-      'updated_at',
-      'student_address',
-      'hispanic_latino',
-      'race'
-    ])
+  def flat_row_hash(student, all_service_types, school)
+    if school.school_type === "HS" then
```

Comment:
This is a tricky one -- since this is Ruby code the triple equals isn't the same as the JS triple equals, use double equals to mean equality in Ruby. https://stackoverflow.com/questions/3422223/vs-in-ruby

Reply:
Thanks! I can never remember which, when, and why.

Reply:
As a rule of thumb I always default to `===` in JavaScript and `==` in Ruby.